data/schema/schema_template.yaml

```diff
@@ -16,24 +16,25 @@ endpoint_url: URL|URL[] # use and array of URLs if multiple files have been down
 derived_from: string[] # OPTIONAL - list of indicator keys this indicator derives from
 framework_group: string # key for the framework group that contains this indicator (from data/aux/framework.json)
 is_experimental: boolean # whether this indicator is experimental
-order: [year, nuts_id, nuts_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
+order: [year, <region_type>_id, <region_type>_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
+query: string # OPTIONAL - the query we POST-ed (not needed for FETCH), e.g. `{city: 'London'}` in the POST example above, or the query sent to  ElasticSearch, etc.
```

Comment:
Is this the query that was used to obtain the original data?

Reply:
yes